Doc/library/trace.rst

```diff
@@ -188,18 +188,20 @@ Programmatic Interface
       Merge in data from another :class:`CoverageResults` object.
 
    .. method:: write_results(show_missing=True, summary=False, coverdir=None,\
-                             ignore_missing_files=False)
+                             *, ignore_missing_files=False)
 
       Write coverage results.  Set *show_missing* to show lines that had no
       hits.  Set *summary* to include in the output the coverage summary per
       module.  *coverdir* specifies the directory into which the coverage
       result files will be output.  If ``None``, the results for each source
       file are placed in its directory.
 
-      If *ignore_missing_files* is True, coverage counts for files that no
+      If *ignore_missing_files* is ``True``, coverage counts for files that no
```

Comment:
And what if it is not `True`, but other true value, like `1` or `'off'`?

Reply:
Yes, but usually the documentation says "if something is true" if not only `bool` is accepted and "if something is ``` ``True`` ```" if it should be literal True. At least it was a convention several years ago. I do not know whether it was written or not.

This is why Victor suggested such correction.

Reply:
I'll keep this in mind in the future.